README.md

```diff
@@ -55,6 +55,8 @@ $ cd ../extension
 $ make install  # no sudo required!
 ```
 
+restart for changes to take effect.
```

Comment:
Please make this start with a capital letter, i.e. `Restart`. This should also clearly state *what* should be restarted. Did you mean `Restart Gnome Shell`?

Reply:
I did a full system restart, but not sure if gnome shell restart is sufficient.
I'll change it to a full restart

Reply:
Ok, I also do not know what should be restarted :) So, restart PC is accepted.